EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80272
+secureblitz.com#?#.widget_text > .widget-title > span:contains(/^ADVERTISEMENT|Advertisement$/):upward(2)
```

Comment:
If possible, better without `:upward`. These rules are slower in times.
`secureblitz.com##.text-7` also can be used on this site, because useful elements has different names(`nav_menu-3` etc)

Reply:
There is another ad below. Check.